client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
Can you move `share-template` into `index.tsx`, there isn't a complicate pieces in it, and moving them away provide hazard more than comfort.

Reply:
I can move to index.tsx, By doing that you are compromising on TDD unit tests and scalability of code, 

If i merged to index.tsx. then i can't write proper unit test cases, it redirects to code scalability reduced. if we start extend this more features in future.

Reply:
True, but the issue doesn't require any additional feature to it 👍, I will leave the discussion to @ahmadabdolsaheb, because he knows the roadmap better.

Reply:
There are pros and cons for both scenarios. We can keep it as is for now.